dashboard/src/app/(authenticated)/(diagram)/components/NodeTypes/Sleep/SleepDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import { NodeRun, SleepNode } from 'littlehorse-client/proto'
+import { FC } from 'react'
+import { NodeDetails } from '../NodeDetails'
+import { formatDate } from '../../../../../utils'
+import { renderVariableAssignment } from './sleepMeta'
+
+export const SleepDetails: FC<{ sleepNode?: SleepNode; nodeRun?: NodeRun }> = ({ sleepNode, nodeRun }) => {
+  if (!sleepNode) return
+
+  return (
+    <NodeDetails>
+      <div className="mb-2">
+        <div className="flex items-center gap-1 whitespace-nowrap text-nowrap">
+          <h3 className="font-bold">SleepNode</h3>
+        </div>
+        <div className="flex flex-col gap-2 text-nowrap">
+          <div className="flex ">
+            {sleepNode.rawSeconds && <div>{renderVariableAssignment(sleepNode.rawSeconds)}</div>}
+            {sleepNode.timestamp && <div>{renderVariableAssignment(sleepNode.timestamp)}</div>}
+            {sleepNode.isoDate && <div>{renderVariableAssignment(sleepNode.isoDate)}</div>}
```

Comment:
This should use the function that already exists as explained in my above comment